pkg/skaffold/schema/validation/validation.go

```diff
@@ -726,30 +726,25 @@ func validateLogPrefix(cfg *parser.SkaffoldConfigEntry, lc latest.LogsConfig) []
 // validateVerifyTests
 // - makes sure that each test name is unique
 // - makes sure that each container name is unique
-func validateVerifyTests(cfg *parser.SkaffoldConfigEntry, tcs []*latest.VerifyTestCase) (cfgErrs []ErrorWithLocation) {
+func validateVerifyTests(runCtx *runcontext.RunContext) []error {
+	var errs []error
 	seenTestName := map[string]bool{}
 	seenContainerName := map[string]bool{}
-	for i, tc := range tcs {
+	tcs := []*latest.VerifyTestCase{}
+	for _, pipeline := range runCtx.GetPipelines() {
+		tcs = append(tcs, pipeline.Verify...)
+	}
+	for _, tc := range tcs {
 		if _, ok := seenTestName[tc.Name]; ok {
-			return []ErrorWithLocation{
-				{
-					Error:    fmt.Errorf("found duplicate test name '%s' in 'verify' test cases. 'verify' test case names must be unique", tc.Name),
-					Location: cfg.YAMLInfos.Locate(&cfg.Verify[i]),
-				},
-			}
+			errs = append(errs, fmt.Errorf("found duplicate test name '%s' in 'verify' test cases. 'verify' test case names must be unique", tc.Name))
```

Comment:
is it possible to get the file name if these verify stanzas are from different `skaffold.yaml` files?

Reply:
I don't believe this information is available in `validation.go` for `skaffold.yaml` files.  I don't think we have the rel/abs path of the SkaffoldConfig(s) stored in any objects we pass around.  Only the flag runCtx.ConfiguraionFile() I believe has something close.  Are you familiar w/ a place this is stored?